### PR TITLE
[WebNN Ep] Slice's axes and steps inputs should be constant initializers

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@fdwr, @guschmue, PTAL, thanks!

File: onnxruntime/core/providers/webnn/builders/impl/slice_op_builder.cc [hunk 1]
```diff
@@ -114,6 +114,22 @@ bool SliceOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
   if (!GetShape(*input_defs[0], input_shape, logger)) {
     return false;
   }
+
+  if (input_defs.size() < 3) {
+    LOGS(logger, VERBOSE) << op_type << " [" << name << "] requires at least 3 inputs (data starts and ends) but got "
```
Comment (fdwr) [inline onnxruntime/core/providers/webnn/builders/impl/slice_op_builder.cc:119]:
```suggestion
    LOGS(logger, VERBOSE) << op_type << " [" << name << "] requires at least 3 inputs (data, starts, ends) but got "
```

**Review by fdwr — APPROVED** [review]

**Review by fdwr — APPROVED** [review]
👍

**Review by guschmue — APPROVED** [review]

**Pull request merged**